README.md

```diff
@@ -38,6 +38,7 @@ This showcases are build around the Bitmovin Adaptive Streaming Player, demonstr
    * [separatedSubtitleAudioSettings](playerUi/separatedAudioSubtitleSettings.html): Extracted audio tracks and subtitle settings from the settings panel direct into the controlBar
    * [labeling](playerUi/labeling.html): Change Languages, Quality and Subtitles labels
    * [customForwardRewindButtons](playerUi/customForwardRewindButtons.html): Custom Forward & Rewind Buttons
+   * [playbackControlsUIWithImaAds](playerUi/playbackControlsUIWithImaAds.html) : Adding playback controls UI during Ads playback with IMA SDK
```

Comment:
Should be `playbackControlsUiWithImaAds` for consistency

Reply:
done